assets/sass/utilities/_utilities-text.scss

```diff
@@ -22,10 +22,7 @@
     h3,
     h4,
     h5,
-    h6 {
```

Comment:
This is a global change but this style was a little unexpected and mean that `u-constrained-content` and `u-constrained-content-wide` now behave the same (with different max-widths)

Reply:
Makes sense – I saw this the other day and was a bit confused by it:

![image](https://user-images.githubusercontent.com/394376/72542248-b9f20400-387b-11ea-8285-233032b1170e.png)